shell/packages/sandstorm-db/db.js

```diff
@@ -590,7 +592,7 @@ isSignedUpOrDemo = function () {
   return false;
 }
 
-var calculateReferralBonus = function(accountId, plan) {
+var calculateReferralBonus = function(user, plan) {
```

Comment:
I think the answer is that we're currently enforcing the wrong quota, but displaying the right one.